features/gateway_injection.md

```diff
@@ -0,0 +1,146 @@
+# Feature: Gateway Injection
```

Comment:
Can you please add this to [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml)?